openapi/go.xml

```diff
@@ -0,0 +1,54 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.kubernetes</groupId>
+    <artifactId>client-python</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <name>client-python</name>
+    <url>http://kubernetes.io</url>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.swagger</groupId>
+                <artifactId>swagger-codegen-maven-plugin</artifactId>
+                <version>${swagger-codegen-version}</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <inputSpec>${generator.spec.path}</inputSpec>
+                            <language>go</language>
+                            <gitUserId>kubernetes-client</gitUserId>
+                            <gitRepoId>go</gitRepoId>
+                            <configOptions>
+                                <packageName>${generator.package.name}</packageName>
+                                <packageVersion>${generator.client.version}</packageVersion>
+                                <sortParamsByRequiredFlag>true</sortParamsByRequiredFlag>
```

Comment:
nit: it seems to [default to true](https://github.com/swagger-api/swagger-codegen/blob/1947220159a2b925f98d06825cf4b376d1cbac13/modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java#L99)